DLaB.CrmSvcUtilExtensions/OptionSet/FilterOptionSetEnums.cs

```diff
@@ -1,200 +0,0 @@
-﻿// =====================================================================
```

Comment:
This file is still used for backwards compatibility.    It shouldn't be deleted.